pages/index.js

```diff
@@ -1,15 +1,57 @@
 import axios from "axios"; // Requests wrapper
 import dayjs from "dayjs"; // Dayjs
-import { useState } from "react"; // State management
+import { useContext, useState } from "react"; // State management
 import Layout from "components/layout"; // Layout wrapper
 import APICTA from "components/api_cta"; // API CTA
 import { web3p, vote } from "containers"; // Context
 import styles from "styles/page.module.scss"; // Page styles
 import BeatLoader from "react-spinners/BeatLoader"; // Loading state
+import { Embedded } from "containers"; // Embedded
 
-export default function Home({ defaultProposals, defaultPages }) {
+
+export default function Home(props) {
+  const [pages, setPages] = useState(props.defaultPages); // Proposal pagination
+  const embedded = useContext(Embedded);
+
+  const proposalsContent = <ProposalsContent {...props} pages={pages} setPages={setPages} />;
```

Comment:
the changes here basically just separate out the view of the recent proposals into its own component, so that we can just show that when embedded as an extension